hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java

```diff
@@ -35,16 +34,16 @@ public JdbcBackendResolver(FQNResolver fqnResolver) {
 
   @Override
   public Optional<Entity> resolveEntity(Event event, StructuredTraceGraph structuredTraceGraph) {
-    if (event.getEventName() != null && event.getEventName().startsWith(JDBC_EVENT_PREFIX) &&
-        SpanAttributeUtils.containsAttributeKey(event, SQL_URL_ATTR)) {
-      String backendUriStr = SpanAttributeUtils.getStringAttribute(event, SQL_URL_ATTR);
+    if (DbTelemetryAttributeUtils.isSqlBackend(event)) {
```

Comment:
curious Q: `isSqlBackend` can return true using non-otel attribute keys, but `getSqlURI` using otel attribute?

Reply:
I would suggest that we don't need `isSqlBackend` explicitly, its functionality can be added to inside `getSqlURi`, what do you think?

Reply:
No, both are consistent. What makes you ask that?